services/skus/appstore.go

```diff
@@ -0,0 +1,308 @@
+package skus
+
+import (
+	"crypto/ecdsa"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+
+	"github.com/awa/go-iap/appstore"
+	"github.com/square/go-jose"
```

Comment:
square jose is deprecated, we should use the maintained version https://github.com/go-jose/go-jose

Reply:
I know. Did not want a side effect from the change until confirming it's working with the existing version.

Would that be acceptable? Trying to minimise the scope of possible errors.

Reply:
Also, performing the upgrade would touch many more other areas, and has to be done carefully and separately.

Reply:
just wanted to point that out, not needed to be addressed immediately

Reply:
I've got a reminder about this in a form of a security issue raised by dependabot. Due to the structure of this repository, the update has to be a separate initiative, which I will carry out once all the P1 work has been addressed in the Premium area.